src/Opdex.Platform.Application/Assemblers/MarketTokenDtoAssembler.cs

```diff
@@ -34,9 +35,18 @@ public async Task<MarketTokenDto> Assemble(MarketToken token)
         }
 
         var tokenAttributes = await _mediator.Send(new RetrieveTokenAttributesByTokenIdQuery(token.Id), CancellationToken.None);
-
         marketTokenDto.Attributes = tokenAttributes.Select(attr => attr.AttributeType);
 
+        var tokenChain = await _mediator.Send(new RetrieveTokenChainByTokenIdQuery(token.Id, false), CancellationToken.None);
+        if (tokenChain is not null)
+        {
+            marketTokenDto.NativeToken = new WrappedTokenDetailsDto
+            {
+                Chain = tokenChain.NativeChain,
+                Address = tokenChain.NativeAddress
+            };
```

Comment:
Might be worth adding to the application mapper profile so this and `TokenDtoAssembler` can share